nixos/modules/installer/cd-dvd/installation-cd-graphical-gnome.nix

```diff
@@ -1,12 +1,12 @@
 # This module defines a NixOS installation CD that contains GNOME.
 
-{ ... }:
+{ lib, ... }:
 
 {
   imports = [ ./installation-cd-graphical-base.nix ];
 
   isoImage.edition = "gnome";
-  isoImage.graphicalGrub = true;
+  isoImage.forceTextMode = lib.mkDefault false;
```

Comment:
Isn't `false` is the default already?

Reply:
Yes, but I'm not changing the semantics that were added here.

AFAIUI this was to keep the fancier menu interface when it had been disabled for the other isos. So in the off-chance it gets disabled again, this will be kept, just like it was.